### PR TITLE
Linear

### PR DESCRIPTION
**Pull request by akhoubani**
I wonder if you could look at the perturbation field initialization.

### CODE REVIEW

File: src/snek5000_cbox/cbox.usr.f [hunk 1]
```diff
@@ -86,6 +86,7 @@ subroutine userchk
 
       integer n, nit_pert, nit_hist
       real Pr_,Ra_
+      real vtmp(lx1*ly1*lz1*lelt,ldim),ttmp(lx1*ly1*lz1*lelt)
```
Comment (ashwinvis) [inline src/snek5000_cbox/cbox.usr.f:89]: You should assign arrays to a common block to persist arrays. You can reuse Nek5000 "scratch" common blocks as done here:

https://github.com/KTH-Nek5000/KTH_Toolbox/blob/a821507125431a091ec0ba28932440de77c2cb72/tools/stat/stat.f#L760-L781

Comment (akhoubani) [reply src/snek5000_cbox/cbox.usr.f:89]: Done!

**Comment by paugier** [comment]
Sorry for the delay. It starts to be in a better shape. I fixed the tests. 

Now doc/examples/run_linear_simple.py need to be turned into a proper unittest (in a file `tests/test_linear.py`). See how it is done in `tests/test_slow.py`. The test in `tests/test_linear.py` should be as short as possible (in particular smallest possible size). It's better if the test actually tests some behavior (as with the `assert` in https://github.com/exabl/snek5000-cbox/blob/main/tests/test_slow.py#L74).

**Comment by paugier** [comment]
It would be also nice to have `doc/examples/run_linear_simple.py` running smaller and shorter simulations. It's really too long now.

**Comment by akhoubani** [comment]
> It would be also nice to have `doc/examples/run_linear_simple.py` running smaller and shorter simulations. It's really too long now.

I'll do it. I also added the scripts for linear simulation to submit the job on the cluster.

**Comment by akhoubani** [comment]
The linear test is too long. I don't know how to skip it with running slow.

**Comment by paugier** [comment]
It's nearly good :-)

When I try `python run_linear_simple.py`, I get:

```
Starting time loop ...

     DT/DTCFL/DTFS/DTINIT   0.120E-01   0.120E-01   0.000E+00   0.500E+00
Step      1, t= 1.1993193E-02, DT= 1.1993193E-02, C=  0.500 0.0000E+00 0.0000E+00
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpirun noticed that process rank 1 with PID 0 on node meige8pcpa79 exited on signal 11 (Segmentation fault).
```

**Comment by akhoubani** [comment]
> It's nearly good :-)
> 
> When I try `python run_linear_simple.py`, I get:
> 
> ```
> Starting time loop ...
> 
>      DT/DTCFL/DTFS/DTINIT   0.120E-01   0.120E-01   0.000E+00   0.500E+00
> Step      1, t= 1.1993193E-02, DT= 1.1993193E-02, C=  0.500 0.0000E+00 0.0000E+00
> --------------------------------------------------------------------------
> Primary job  terminated normally, but 1 process returned
> a non-zero exit code. Per user-direction, the job has been aborted.
> --------------------------------------------------------------------------
> --------------------------------------------------------------------------
> mpirun noticed that process rank 1 with PID 0 on node meige8pcpa79 exited on signal 11 (Segmentation fault).
> ```

I don't get this error. Could you please try `resources={"nproc": 2}` to see if it is related to the available processors?

**Comment by paugier** [comment]
Should we do something with this PR? I guess we could remove this branch? What do you think @akhoubani ?

**Comment by paugier** [comment]
Otherwise we need to rebase on master.

**Comment by akhoubani** [comment]
Yes, it would be nice to rebase it. I wanted to add scripts for linear sims on the cluster.

**Comment by akhoubani** [comment]
@paugier This can be merged.

**Pull request merged**